src/ExpandObjects/epfilter.f90

```diff
@@ -17430,6 +17438,7 @@ SUBROUTINE CreateNewUnitary
     CALL AddToObjFld('Speed 1 Gross Rated Heating COP {W/W}', base + ussHPHeatCoilCOPOff,' ')
     CALL AddToObjStr('Speed 1 Rated Air Flow Rate {m3/s}','autosize')
     CALL AddToObjStr('Speed 1 Rated Supply Air Fan Power Per Volume Flow Rate {W/(m3/s)}','')
+    CALL AddToObjStr('Speed 1 2023 Rated Evaporator Fan Power Per Volume Flow Rate {W/(m3/s)}','')
```

Comment:
And for heating coils "Rated Supply Air Fan Power . . ." (not Evaporator). And it looks like some dx heating coils may have been missed?

Reply:
I think it's good to go now.  I believe it is working properly in all the instances of `CALL CreateNewObj('CoilTypeHere')`